### DISCUSSION TITLE
Text area max height

### DISCUSSION THREAD

**Discussion by PabloFNK** [twilio-labs/paste#1672] [Q&A] [answered]
TextArea has a fixed max height after what it displays a scroll bar. Is there a way to change that to display the scrollbar with less lines?

**Comment by TheSisb** [comment, marked as answer]
Hi @PabloFNK,

There's currently no way to change that. Can you provide an example of what you're trying to do so we can take it into consideration for a possible change to the component? I can tell you that we designed the textarea this way to be consistent and visually different from text inputs.
